pyproject.toml

```diff
@@ -1,13 +1,14 @@
 [tool.poetry]
 name = "pandantic"
 version = "0.1.2"
-description = ""
-authors = ["wessel.huising <wessel.huising@mollie.com>"]
+description = "Validate various dataframes like Pandas using the Pydantic API"
```

Comment:
I think we can improve this description grammatically. I also think the filter functionality should be made more explicit.

How about:
* "Validate and filter various dataframes (i.e., pandas, etc.) using Pydantic."
* "Leverage Pydantic to validate or filter your dataframe."
* Or a fun one: "Type safe code is good code: validate and filter your dataframes with Pydantic!"
* Another more fun one: "Gone are the days of black-box dataframes in otherwise type-safe code! Pandantic builds off the Pydantic API to enable validation and filtering of dataframes (i.e., pandas, etc.) "

You can come up with your own but I think you get the jist.

Reply:
I really like the 4th one.